frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -421,409 +423,378 @@ class BitBox02 extends Component<Props, State> {
-                                        <div className="columnsContainer m-top-default">
-                                            <div className="columns">
-                                                <div className="column column-1-2">
-                                                    <div className={style.stepContext} style={{minHeight: 330}}>
-                                                        <h3 className={style.stepSubHeader}>{t('button.create')}</h3>
-                                                        <p className="text-center">{t('bitbox02Wizard.stepUninitialized.create')}</p>
-                                                        <div className={['buttons text-center', style.fullWidth].join(' ')}>
-                                                            <Button
-                                                                primary
-                                                                onClick={this.createWalletStep}
-                                                                disabled={settingPassword}>
-                                                                {t('seed.create')}
-                                                            </Button>
-                                                        </div>
-                                                    </div>
-                                                </div>
-                                                <div className="column column-1-2">
-                                                    <div className={style.stepContext} style={{minHeight: 330}}>
-                                                        <h3 className={style.stepSubHeader}>{t('button.restore')}</h3>
-                                                        <p className="text-center">{t('bitbox02Wizard.stepUninitialized.restore')}</p>
-                                                        <div className={['buttons text-center', style.fullWidth].join(' ')}>
-                                                            <Button
-                                                                primary
-                                                                onClick={this.restoreBackupStep}>
-                                                                {t('bitbox02Wizard.stepUninitialized.restoreMicroSD')}
-                                                            </Button>
-                                                            <Button
-                                                                primary
-                                                                onClick={this.restoreFromMnemonic}>
-                                                                {t('bitbox02Wizard.stepUninitialized.restoreMnemonic')}
-                                                            </Button>
-                                                        </div>
-                                                    </div>
```

Comment:
The View component did not have a way of displaying content in 2 columns / cards next to each other. Added Grid/Columns component with asCard prop to render as card.